### PR TITLE
Workaround for getting debuginfo from Media3 repo

### PR DESCRIPTION
**Pull request by mnowaksuse**
zypper has a feature which rewrites last digit in repo's URI to '1'
unless it's '1' already. We need to alter name for that repository
when rsyncing it.

Ref:
https://mailman.suse.de/mlarch/SuSE/research/2016/research.2016.07/msg00111.html

Verification run: http://assam.suse.cz/tests/1682#step/crash/28.

### CODE REVIEW

File: tests/toolchain/crash.pm [hunk 1]
```diff
@@ -44,7 +44,11 @@ sub run() {
         my $url = "ftp://openqa.suse.de/" . get_var('REPO_SLES_DEBUG');
         assert_script_run "zypper ar -f $url SLES-Server-Debug";
 
-        assert_script_run 'zypper ref; zypper -n -v in kernel-default-base-debuginfo kernel-default-debuginfo', 300;
+        # workaround before POOL metadata in Media3 are fixed
```
Comment (okurz) [inline tests/toolchain/crash.pm]: don't forget to record a soft failure with a reference to a bug for every "product workaround" we have in code.

**Comment by sysrich** [comment]
Any news from Rudi (aka bugfinder in IRC) about fixing the actual meta data so this workaround is unnecessary?

**Comment by mnowaksuse** [comment]
@sysrich It's just a unexpected zypper feature. See updated commit msg.

**Comment by sysrich** [comment]
Ok, in the light of the new info, I'd like to see the rsync.pl do the renamining of Media3 to DebugInfo, then we can use that

**Comment by sysrich** [comment]
According to what I've read on mailinglists, the new version of zypper in SP2 no longer has this behaviour..so we have nothing to do.. @mnowaksuse do you agree, or do you want us to do something? (I don't want to close this PR while you're on vacation)

**Comment by mnowaksuse** [comment]
Agreed. From what I've seen the original problem is fixed.

**Pull request closed without merging**